langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -64,6 +89,11 @@ public isolated function getCodePoint(string str, int index) returns int = @java
 
 # Returns a substring of a string.
 #
+# ```ballerina
+# "Hello, my name is John".substring(7) ⇒ my name is John
+# "Hello, my name is John Anderson".substring(18, 22) ⇒ John
```

Comment:
```suggestion

// Retrieve the substring between 18(inclusive) and 22 indexes
# "Hello, my name is John Anderson".substring(18, 22) ⇒ John
```

Reply:
@pcnfernando Isn't the details given in the comment covered by the description of the `return`.

Reply:
Yes, true. But I thought explaining it in the example itself will be useful. wdyt?

Reply:
Actually, we eliminated such comments which are already covered by the description and etc. I believe let's avoid this just to keep it synced with other such example.